src/derivatives/atlas.md

```diff
@@ -472,6 +473,141 @@ A guide for using macros can be found at
 })
 }}
 
+**Producing a new template AND atlas.**
+Atlasing is often performed with reference to a *custom* standard space.
+In this case, a feature template map is generated from all the participant(s)
+in the study, and the atlas' artifacts are produced with reference to that
+template.
+
+Either by generating the template space with aligning to a pre-existing template,
+or by estimating a transform between templates by means of image registration,
+a new template definition MUST be employed if the new template generates
+a new [*space*](../common-principles.md#definitions).
+For example, let's imagine that PS13 first generated a template nuclear imaging
+map and after that, a corresponding [*atlas*](../common-principles.md#definitions)
+was defined.
+In that case, the [`atlas-<label>`](../glossary.md#atlas-entities) SHOULD be omitted
+except several atlases need specification:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "ps13-pipeline": {
+      "tpl-PS13": {
+         "pet": {
+            "tpl-PS13_desc-nopvc_dseg.nii.gz": "",
+            "tpl-PS13_desc-pvc_dseg.nii.gz": "",
+            "tpl-PS13_dseg.json": "",
+            "tpl-PS13_dseg.tsv": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.nii.gz": "",
```

Comment:
In reality this would rarely happen without the definition of "_space-fsaverage", aka an anatomical reference space. So maybe adding  "_space-" entities in these examples to make this more realistic? (We do understand that it is possible, but is not what most people would do.)
Also in order to keep the two templates for the different spaces in the same folder, we need to have "_space-fsaverage" and  "_space-MNI152Lin" to disambiguate.

Reply:
Responded below -- if I understand correctly, I think this is a duplicate comment right? https://github.com/bids-standard/bids-specification/pull/1856#discussion_r1794116590